templates/watcher/config/00-default.conf

```
@@ -52,9 +52,20 @@ auth_type = password
 lock_path = /var/lib/watcher/tmp
 
 [watcher_datasources]
+# This must be changed when we have prometheus datasource merged
 datasources = ceilometer
 
 {{ if (index . "MemcachedServers") }}
 [cache]
 memcached_servers = {{ .MemcachedServers }}
 {{ end }}
+
+{{ if (index . "PrometheusHost") }}
+[prometheus_client]
+host = {{ .PrometheusHost }}
+port = {{ .PrometheusPort }}
```

Can we also move PrometheusPort under conditional?
```suggestion
{{ if (index . "PrometheusPort") }}
port = {{ .PrometheusPort }}
{{ end }}
```